userapi/src/main/java/fk/prof/userapi/http/UserapiApiPathConstants.java

```diff
@@ -1,13 +1,31 @@
 package fk.prof.userapi.http;
 
 public final class UserapiApiPathConstants {
+    private UserapiApiPathConstants() {
+    }
 
-  private UserapiApiPathConstants() { }
+    public static final String POLICY_API_PREFIX = "/policy";
+    public static final String LIST_POLICY_API_PREFIX = "/list/policy";
+    public static final String DELIMITER = "/";
```

Comment:
Preferably declared on the top. Also, used anywhere?

Reply:
Was used in test class, moved there.